### PR TITLE
Skip Gantt plots with non-fusion runs

### PR DESCRIPTION
**Pull request by pinin4fjords**
Addresses https://github.com/seqeralabs/nf-aggregate/issues/18

- [ ] TODO: add a non-fusion run ID to test data

### CODE REVIEW

**Review by maxulysse — APPROVED** [review]

File: modules/local/seqera_runs_dump/main.nf [hunk 1]
```diff
@@ -28,7 +29,7 @@ process SEQERA_RUNS_DUMP {
         -id=${meta.id} \\
         --workspace=${meta.workspace} \\
         --output="${prefix}.tar.gz" \\
-        $args2
+        $args2 $fusion
```
Comment (drpatelh) [inline modules/local/seqera_runs_dump/main.nf:32]:
```suggestion
        $fusion \\
        $args2
        
```

File: workflows/nf_aggregate/main.nf [hunk 2]
```diff
@@ -34,15 +34,20 @@ workflow NF_AGGREGATE {
     )
     ch_versions = ch_versions.mix(SEQERA_RUNS_DUMP.out.versions.first())
 
+    ch_runs = SEQERA_RUNS_DUMP.out.run_dump
+        .branch {
+            gantt: it[0].fusion && ! params.skip_run_gantt
+            non_gantt: (! it[0].fusion) || (params.skip_run_gantt)
+        }
```
Comment (drpatelh) [inline workflows/nf_aggregate/main.nf:41]:
```suggestion
    SEQERA_RUNS_DUMP
        .out
        .run_dump
        .branch {
            meta, run_dump ->
                gantt: meta.fusion && !params.skip_run_gantt
                no_gantt: !meta.fusion || params.skip_run_gantt
        }
        .set { ch_runs }
```

File: modules/local/seqera_runs_dump/nextflow.config [hunk 3]
```diff
@@ -1,11 +1,11 @@
 process {
     withName: 'SEQERA_RUNS_DUMP' {
         ext.args = { params.seqera_cli_extra_args ? params.seqera_cli_extra_args.split("\\s(?=--)") : '' }
-        ext.args2 = { params.skip_run_gantt ? '' : '--add-task-logs --add-fusion-logs' }
+        ext.args2 = { params.skip_run_gantt ? '' : '--add-task-logs' }
```
Comment (drpatelh) [inline modules/local/seqera_runs_dump/nextflow.config:4]: Can't remember why we added `--add-task-logs` here when using `params.skip_run_gantt`. Do we need to move this setting into the process too?

File: workflows/nf_aggregate/main.nf (hunk 2 above)
Comment (drpatelh) [reply workflows/nf_aggregate/main.nf:41]: Do we need to `branch` if we aren't using `no_gantt` anywhere? Could just use a `filter` instead.

Comment (pinin4fjords) [reply workflows/nf_aggregate/main.nf:41]: This was me thinking of things expanding in future, but OK, will switch to filter.

File: modules/local/seqera_runs_dump/nextflow.config (hunk 3 above)
Comment (pinin4fjords) [reply modules/local/seqera_runs_dump/nextflow.config:4]: No, don't think so, we'd have to pass `params.skip_run_gantt` into the process, which doesn't seem sensible. 

The other option was switched to output the fusion logs depending on fusions status, which is native to the task, which this is not.

**Review by maxulysse — APPROVED** [review]

**Comment by pinin4fjords** [comment]
Thanks for review @drpatelh @maxulysse

**Pull request merged**